src/utils/helper.js

```diff
@@ -0,0 +1,19 @@
+// Based on https://stackoverflow.com/a/43478439/513672
+const arraysEquality = (_arr1, _arr2) => {
```

Comment:
Could you just use lodash's isEqual here?

https://lodash.com/docs/4.17.10#isEqual

Supposedly it supports arrays

Reply:
I'm not too familiar with `lodash`. Thanks for pointing it out to me.